packages/app/src/app/pages/Sandbox/Editor/Workspace/Advertisement/Advertisement.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { useScript } from 'app/hooks';
+import { Container } from './elements';
+
+export const Advertisement = () => {
+  useScript(`https://codefund.app/properties/24/funder.js`);
```

Comment:
You can just use normal quotes here

```suggestion
  useScript('https://codefund.app/properties/24/funder.js');
```

Reply:
Actually going to keep this one as-is. While yes in this case we're not making use of template literals, there's no cost to using backticks as opposed to quotes and that being the case it's just a good practice to prefer backticks for the simple reason that you can at any time drop in expressions and mixed quotes at any time.

This article sold me on template literals being strictly better a while back and so I've been using them everywhere since:

https://ponyfoo.com/articles/template-literals-strictly-better-strings